examples/react/css/src/common.css

```diff
@@ -59,3 +59,18 @@ input[class^='ais-']:focus-visible {
   white-space: nowrap;
   font-size: 0.75em;
 }
+
+input[class^='ais-'][type='checkbox']:checked {
+  background-image: url("data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20fill%3D%22none%22%20aria-hidden%3D%22true%22%20viewBox%3D%220%200%2016%2012%22%3E%3Cpath%20stroke%3D%22%23fff%22%20stroke-linecap%3D%22round%22%20stroke-linejoin%3D%22round%22%20stroke-width%3D%223%22%20d%3D%22M1%205.917%205.724%2010.5%2015%201.5%22%2F%3E%3C%2Fsvg%3E");
+  background-repeat: no-repeat;
+  background-size: var(--ais-spacing-half);
```

Comment:
FYI I removed it in my searchbox PR, I thought we might want to avoid computed variables. WDYT?

Reply:
Yep, didn't have time to rework it on my PRs

Reply:
I did it in #6584, I'll rebase this one and update it too